go/common/enclave.go

```diff
@@ -100,11 +99,11 @@ type Enclave interface {
 	// Subscribe adds a log subscription to the enclave under the given ID, provided the request is authenticated
 	// correctly. The events will be populated in the BlockSubmissionResponse. If there is an existing subscription
 	// with the given ID, it is overwritten.
-	Subscribe(id uuid.UUID, encryptedParams EncryptedParamsLogSubscription) error
+	Subscribe(id rpc.ID, encryptedParams EncryptedParamsLogSubscription) error
```

Comment:
With the new way I create the subscription, I can just reuse the Geth subscription ID everywhere, instead of creating an additional UUID.